Makefile

```diff
@@ -35,3 +35,7 @@ ocp_cleanup:
 
 host_cleanup:
 	./host_cleanup.sh
+
+bell:
+	echo "Done!"
+	tput bel
```

Comment:
Is this different than just echoing \a?

Reply:
It's not a big deal, was just wondering if we needed to pull in ncurses, but it's probably already installed on everything anyway.  It's in `@core` package group.

Reply:
Probably not. Some of the tips I found implied this worked better from within screen.